### PR TITLE
Add SideShift.ai exchange plugin

### PR DESCRIPTION
**Pull request by abrkn**
This pull request adds necessary metadata and logos for [SideShift.ai](https://sideshift.ai) functionality to Edge Wallet.

`affiliateId` can be obtained from the [SideShift.ai affiliate page](https://sideshift.ai/Affiliate).

#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [x] Tested on iOS Tablet
- [x] Tested on small Android

The necessary JSON field is added to the `env` file, the plugin is initialized and the required SideShift.ai logos are added as described in the swap plugin integration [Wiki page]( https://github.com/EdgeApp/edge-react-gui/wiki/Integration-of-a-swap-plugin).

Accompanying pull requests: 
- [edge-exchange-plugins](https://github.com/EdgeApp/edge-exchange-plugins/pull/100)
- [edge-reports](https://github.com/EdgeApp/edge-reports-server/pull/81)

### CODE REVIEW

**Review by thehobbit85 — APPROVED** [review]
Waiting for this PR to be fixed and merged before merging this:
https://github.com/EdgeApp/edge-exchange-plugins/pull/100

**Comment by thehobbit85** [comment]
@paullinator, This is ready to be merged after the "develop" branch is out of "freeze" mode.

**Comment by abrkn** [comment]
> @paullinator, This is ready to be merged after the "develop" branch is out of "freeze" mode.

Is it still in freeze? @thehobbit85 @paullinator

**Pull request merged**
